OasisDEX/oasis-borrow
# Network switcher fixes

# [Network switcher fixes](https://app.shortcut.com/oazo-apps/story/8644/network-switcher-fixes)

## Changes 👷‍♀️
- styling fixes (tick position, hover animation)
- synced settings when the network is forced (eg. on reconnect), this fixed the blicking connect button
- added a redirection of user closes the connection modal on a page where connection is needed (or disconnects on such page)

## How to test
- the network switcher should look better, should have hover effects
- the errors where you access a page which requires a wallet connection (eg. `vaults/open-multiply/WSTETH-A`) with no wallet connected should be gone
- basically everything should be fine just fine :v:
